crates/agent/src/connector_tags.rs

```diff
@@ -117,15 +123,20 @@ impl TagHandler {
             }
         }
 
-        let proto_type = match runtime::flow_runtime_protocol(&image_composed).await {
-            Ok(ct) => ct,
-            Err(err) => {
-                tracing::warn!(image = %image_composed, error = %err, "failed to determine connector protocol");
-                return Ok((row.tag_id, JobStatus::SpecFailed));
+        let proto_type = if row.image_name.starts_with(DEKAF_IMAGE_NAME_PREFIX) {
```

Comment:
I kinda feel like this conditional should be in `runtime::flow_runtime_protocol`. Is there a reason to prefer it to be in `agent`?

Reply:
Yeah, you're right. I read `runtime::flow_runtime_protocol` and `parse_image_inspection` briefly and saw that `docker inspect` call, but it's trivial to check the image first. :+1: